src/toggle-attribute/README.md

```diff
@@ -64,13 +64,20 @@ import '@travelopia/web-components/dist/toggle-attribute/style.css';
 		<option value="">Select</option>
 		<option value="First">First</option>
 		<option value="Second">Second</option>
+		<option value="Third">Third</option>
+		<option value="Fourth">Fourth</option>
+		<option value="Fifth">Fifth</option>
 		<option value="All">All</option>
 	</select>
 </tp-toggle-attribute>
 
 <div class="toggle-target-variable" data-toggle-value="First,All">
 	Toggled First
 </div>
+<div class="toggle-target-variable" data-toggle-value-non-empty>
```

Comment:
@abhishekxix The attribute is expected to be added to the web component, not the target

Reply:
@junaidbhura, would it be a good idea to keep it on both for flexibility as we have done for other cases in this component?

Reply:
@abhishekxix Could you elaborate what flexibility this will offer?

Reply:
Sure @junaidbhura. The two approaches will cater to the following two cases.

## Case 1
The attribute is put on `tp-toggle-attribute`

```html
<tp-toggle-attribute target=".toggle-target-non-empty" value-non-empty="yes">
	<select>
		<option value="">Select</option>
		<option value="First">First</option>
		<option value="Second">Second</option>
		<option value="Third">Third</option>
		<option value="Fourth">Fourth</option>
		<option value="Fifth">Fifth</option>
		<option value="All">All</option>
	</select>
</tp-toggle-attribute>

<!-- Every target will be toggled for every non empty value on the trigger. -->
<div class="toggle-target-non-empty"> 
	Toggle for all non empty values.
</div>

<div class="toggle-target-non-empty"> 
	Toggle for all non empty values 2.
</div>

```

## Case 2
The attribute is put on target
```html
<tp-toggle-attribute target=".toggle-target-variable">
	<select>
		<option value="">Select</option>
		<option value="First">First</option>
		<option value="Second">Second</option>
		<option value="Third">Third</option>
		<option value="Fourth">Fourth</option>
		<option value="Fifth">Fifth</option>
		<option value="All">All</option>
	</select>
</tp-toggle-attribute>

<!-- This will be toggled only for the values provided -->
<div class="toggle-target-variable" data-toggle-value="First,All">
	Toggled First
</div>
<!-- This will be toggled for every non empty value on the trigger -->
<div class="toggle-target-variable" data-toggle-value-non-empty="yes">
	This will be toggled for all non empty values.
</div>
<div class="toggle-target-variable" data-toggle-value="Second,All">
	Toggled Second
</div>
```

## Conclusion
- Use the attribute on the trigger when all targets should be toggled for all non empty trigger values.
- Use the attribute on a target to still keep the other targets behaving as they were according to values specified on them, but toggle this one target on every non empty value.